src/rules/bn.toml

```diff
@@ -0,0 +1,39 @@
+
+# Replacements which are run before sentences are split
+replacements = []
+
+min_trimmed_length = 3
+min_word_count = 3
```

Comment:
Unlikely, but just to make sure: there are no useful sentences with 2 or less words in Wikipedia. Right?

Reply:
It's rare, tough there can be meaningful phrases as well as entities which can make sense. I will try out with 2 min words and check if that add reasonable words in the data.